screen/setup_mae_from_com.py

```diff
@@ -51,14 +51,14 @@ def read_com(filename):
             if cols[0] == 'COMP':
                 comp.extend(map(int, cols[1:5]))
             if cols[0] == 'TORS':
-                tors.append(tuple(map(int, cols[1:3])))
+                tors.append(list(map(int, cols[1:3])))
```

Comment:
Wrapping list() around map() works but it's typically better to use list comprehensions as it's considered more clear to read and follows the python standards. Map can be _slightly_ more efficient unless you're using a lambda inside of map in which case list comprehensions are more efficient, but it's such a small difference that it typically won't matter. This line would be better written as
```suggestion
                tors.append([int(x) for x in cols[1:3]])
```